docker-compose.yaml

```diff
@@ -1,22 +1,61 @@
-version: '3'
 services:
+  certs:
+    image: jenkinsciinfra/ldap:latest
+    volumes:
+      - certs:/certs:rw
+    entrypoint:
+      - "/bin/bash"
+      - -eucx
+      - |
+        if test -f /certs/rootCA.pem; then exit 0;fi
+        apt-get update -q
+        apt-get install --yes --no-install-recommends curl
+        curl --silent --show-error --location "https://dl.filippo.io/mkcert/latest?for=linux/$(dpkg --print-architecture)" \
+          --output /usr/local/bin/mkcert
+        chmod +x /usr/local/bin/mkcert
+        mkcert -key-file /certs/privkey.key -cert-file /certs/cert.pem ldap ldap.localhost localhost 127.0.0.1 ::1
+        cp "$(mkcert -CAROOT)"/rootCA.pem /certs/rootCA.pem
+        chown 101 /certs/*
+        chmod 0664 /certs/*
+
   ldap:
-    build: ./mock-ldap
+    image: jenkinsciinfra/ldap:latest
+    depends_on:
+      certs:
+        condition: service_completed_successfully
+    environment:
+      OPENLDAP_SSL_CA_ROOTDIR: /etc/ldap/ssl
+      OPENLDAP_SSL_CA: rootCA.pem
+    volumes:
+      - certs:/etc/ldap/ssl/:ro
     ports:
         - "1636:636"
         - "1389:389"
-  mail:
-    image: rnwood/smtp4dev
-    ports:
-      - "3000:80"
-      - "2525:25"
-      - "1143:143"
```

Comment:
This service was not starting on my tests: I've removed it and we'll see if we need it in a subsequent PR

Reply:
this works perfectly fine for me, macOS arm docker.

Its needed for testing of emails:

<img width="1438" alt="image" src="https://github.com/user-attachments/assets/3cad90ba-7d54-4a95-a1d9-2f440c0b5109" />

Reply:
thanks, I'll retry carefully then

Reply:
it works, I don't know what I did wrong yesterday. it's back (both compose and README)